Rtd/proof output kpi (#878)
rl-institut/multi-vector-simulator: docs/model/simulation_outputs.rst

Economic KPI
############

All the KPI related to costs described below are provided in net present value.

.. include:: outputs/costs_total.inc

The Net present costs (NPC) is the present value of all the costs associated with installation, operation, maintenance and replacement of energy assets within the optimized multi-vector energy system over the whole project lifetime, deducting the present value of the residual values of asset at project and as well as all the revenues that it earns over the project lifetime. The capital recovery factor (CRF) is used to calculate the present value of the cash flows.

.. math::

>     NPC = \sum_i{(c_specific + c_replacement + c_residual) \cdot CAP_i} + \sum_i{\sum_t{E_i(t)} \cdot p_dispatch}

Author: ```suggestion
    NPC = \sum_i{(c_{specific} + c_{replacement} + c_{residual}) \cdot CAP_i} + \sum_i{\sum_t{E_i(t)} \cdot p_{dispatch}}
```

Economic KPI
############

All the KPI related to costs described below are provided in net present value.

.. include:: outputs/costs_total.inc

The Net present costs (NPC) is the present value of all the costs associated with installation, operation,
maintenance and replacement of energy technologies comprising the sector-coupled system over the project lifetime,
minus the present value of all the revenues that it earns over the project lifetime.
The capital recovery factor (CRF) is used to calculate the present value of the cash flows.

> **The content of this section was copied from the** `conference paper <http://doi.org/10.5281/zenodo.4449918>`_ **handed in to CIRED 2020**

Reviewer: I think I would rather reformulate this paragraph. Probably not use cites for now.

.. include:: outputs/levelized_costs_of_electricity_equivalent.inc

> A multi-vector energy system connects energy vectors into a joined, energy system. The optimization objective of the MVS then is to minimize the overall energy costs, without distinguising the different sectors. This sector-coupled energy system is then designed to have an optimial, joined operation. With other systems, the costs associated to each individual energy vector would be used to calculate the levelized costs of energy (LCOEnergy). With the multi-vector system, this could lead to distorted costs - for example if there is a lot of PV (electricity sector), which in the end is only supplying an electrolyzer (H2 sector). The LCOE of electricity would thus turn out to be very high, which could be considered unfair as the electricity from PV is solely used to provide the H2 demand. 

Author: ```suggestion
A multi-vector energy system connects energy vectors into a joined energy system. The optimization objective of the MVS then is to minimize the overall energy costs, without distinguising between the different sectors. This sector-coupled energy system is then designed to have an optimial, joined operation. With other systems, the costs associated to each individual energy vector would be used to calculate the levelized costs of energy (LCOEnergy). With the multi-vector system, this could lead to distorted costs - for example if there is a lot of PV (electricity sector), which in the end is only supplying an electrolyzer (H2 sector). The LCOE of electricity would thus turn out to be very high, which could be considered unfair as the electricity from PV is solely used to provide the H2 demand. 
```

.. include:: outputs/levelized_costs_of_electricity_equivalent.inc

> A sector-coupled system connects energy vectors, not the costs associated to each individual energy carrier. However, the overall energy costs should be minimized. Therefore, we define a new KPI: The levelized costs of energy (LCOEnergy), which aggregates the costs for energy supply and distributes them over the total energy demand supplied. The latter is calculated with weighting factors for each energy carriers. To determine the weighting factors of the different energy carriers, we reference the method of gasoline gallon equivalent (GGE) `[12] <https://epact.energy.gov/fuel-conversion-factors>`_, which enables the comparison of alternative fuels. Instead of comparing the energy carriers of an MES to gasoline, we rebase the factors introduced in `[12] <https://epact.energy.gov/fuel-conversion-factors>`_ onto the energy carrier electricity, thus proposing a unit Electricity Equivalent (ElEq). The necessary weights are summarized in Table 1. With this, we propose to calculate LCOEnergy based on the annual energy demand and the systems annuity, calculated with the CRF, as follows:

Reviewer: ```suggestion
A multi-vector energy system connects energy vectors into a joined, energy system. The optimization objective of the MVS then is to minimize the overall energy costs, without distinguising the different sectors. This sector-coupled energy system is then designed to have an optimial, joined operation. With other systems, the costs associated to each individual energy vector would be used to calculate the levelized costs of energy (LCOEnergy). With the multi-vector system, this could lead to distorted costs - for example if there is a lot of PV (electricity sector), which in the end is only supplying an electrolyzer (H2 sector). The LCOE of electricity would thus turn out to be very high, which could be considered unfair as the electricity from PV is solely used to provide the H2 demand. 
Therefore, we define the :ref:`attributed costs of each energy vector <attributed_costs>`, to determine how much of the overall system costs should be attributed to one sector, depending on the energy demand it has compared to the other sectors. To be able to compare the demands of different energy carriers, :ref:`energy carrier weighting <energy_carrier_weighting>` is applied. 
We then define the the levelized costs of energy (LCOEnergy), based on the annuity of the attributed costs and the demand of one energy sector.  
```
How about this? The first part about the attributed costs could also be moved further up. The details about energy carrier weighting are already explained here: https://multi-vector-simulator.readthedocs.io/en/latest/model/assumptions.html#weighting-of-energy-carriers

> 
> .. TODO: nothing follows as mentionned in the text ...
> 
> .. TODO: add link to Table 1. within readthedocs

Reviewer: ```suggestion
```

Specific electricity supply costs, eg. levelized costs of electricity (LCOElectricity) are a common KPI that can be compared to local prices or generation costs. As in a sector-coupled system the investments cannot be clearly distinguished into sectors, we propose to calculate the levelized costs of energy carriers by distributing the costs relative to supplied demand. The LCOElectricity are then calculated with:

> **The content of this section was copied from the** `conference paper <http://doi.org/10.5281/zenodo.4449918>`_ **handed in to CIRED 2020**

Reviewer: This is partially a repitition of the above, we can partially delete it.

Reviewer: Is the equation of the LCOEnergy really missing?

Reviewer: Fixed in f592ff3f404bc46c54d5e09866eff5d0d13d0b94

.. math:: RF = \frac{ 100 kWh(el)\cdot \frac{kWh(eleq)}{kWh(el)} +50 kWh(el) \cdot \frac{kWh(eleq)}{kWh(el)}}{200 kWh(el) \cdot \frac{kWh(eleq)}{kWh(el)}} = 3/4 = \text{75 \%}

> The renewable factor, just like the :ref:`renewable_share_of_local_generation`, cannot indicate how much renewable energy is used in each of the sectors. It is nevertheless possible to tackle this issue with the degree of sector-coupling.

Reviewer: ```suggestion
The renewable factor, just like the :ref:`renewable_share_of_local_generation`, cannot indicate how much renewable energy is used in each of the sectors. In the future, it might be possible to get a clearer picture of the flows between the sectors with the proposed :ref:`degree of sector-coupling <degree_of_sector_coupling>`.
```

        \text{with } i,j &\text{: Electricity,H2…}

> **The content of this section was copied from the** `conference paper <http://doi.org/10.5281/zenodo.4449918>`_ **handed in to CIRED 2020**

Reviewer: This part should be reformulated

.. image:: images/network_graph.png
 :width: 600

Files
#####

> 

Reviewer: We should rather rename `Figures` to `Files`, as all of them are enumerated as files together at the top of the page:
![grafik](https://user-images.githubusercontent.com/44204527/117445401-eb905200-af3a-11eb-942a-a1853637b2ec.png)

Also, I noticed that we should add "files" to this sentence:
![grafik](https://user-images.githubusercontent.com/44204527/117445447-f9de6e00-af3a-11eb-86fc-4ed4f92c647e.png)

(If you want to distinguish into files and figures, we need to introduce a new kpi kategory in `mvs_kpi_categories.csv`

Reviewer: Do you mean we should use both of the categories, ie. `files` and `figures`?
